ivy/container/base.py

```diff
@@ -3162,7 +3162,7 @@ def map_conts(
     def with_entries_as_lists(self):
         def to_list(x, _=""):
             try:
-                return self._ivy.to_list(x)
+                return x
```

Comment:
are you sure about this? The original seems correct to me.

Reply:
The original throws an error when the container has a string saying the string object can't be used with to_list(). The new version doesn't have this problem. Is there a better way to handle this?